app/presenters/prior_authority/check_answers/base.rb

```diff
@@ -23,7 +23,7 @@ def title(**)
 
       def rows
         row_data.map do |row|
-          row_content(row[:head_key], row[:text], row[:actions], row[:head_opts] || {})
+          row_content(row[:head_key], row[:text], row[:head_opts] || {}, row[:actions] || nil)
```

Comment:
is the row[:actions] `|| nil` bit needed? won't it be nil if not supplied anyway

```suggestion
          row_content(row[:head_key], row[:text], row[:head_opts] || {}, row[:actions])
```